package.json

```diff
@@ -24,7 +24,7 @@
     "typecheck": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false"
   },
   "dependencies": {
-    "@arcana/auth": "^0.1.3",
+    "@arcana/auth": "^0.3.0",
```

Comment:
Revert this to 0.1.3, we'll be using old auth sdk in developer dashboard until NoUI auth sdk is built. Dashboard won't have UI mode for wallet